src/index.html

```diff
@@ -11,10 +11,10 @@
 
 <body>
   <section class="welcome-section">
-      <img src="./images/logo.png" class="fitlit-logo" alt="FitLit logo">
```

Comment:
Thanks for removing this.  It was a cute logo but didn't flow well as we implemented other changes.